pkg/controller/controller_broker_test.go

```diff
@@ -299,22 +288,27 @@ func TestReconcileBrokerErrorFetchingCatalog(t *testing.T) {
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := api.EventTypeWarning + " " + errorFetchingCatalogReason + " " + "Error getting broker catalog for broker \"test-broker\": instance not found"
+	expectedEvent := api.EventTypeWarning + " " + errorFetchingCatalogReason + " " + "Error getting broker catalog for broker \"test-broker\": ooops"
```

Comment:
Also, I'm ok with doing the cleanup in a follow-up, in favor of getting this improvement in as a foundation